_posts/2023-10-02-intellij-quarkus-recap.adoc

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: A recap of Quarkus Tools for IntelliJ's latest improvements
+date: 2023-10-02
+tags: intellij ide
+synopsis: Over the past five months, we’ve been really hard at work to improve Quarkus Tools in every possible way. So let’s look in more details at what was accomplished.
+author: fbricon
+---
+:imagesdir: /assets/images/posts/intellij-quarkus-tools-oct23
+
+https://plugins.jetbrains.com/plugin/13234-quarkus-tools[Quarkus Tools for IntelliJ] is a free and open source extension, helping users develop Quarkus applications by providing content-assist, validation, run configurations and many other features right from their favourite IDE.
+
+This extension is based on the https://github.com/eclipse/lsp4mp[LSP4MP] (i.e. MicroProfile) and its https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls.ext[Quarkus] add-on, and the https://github.com/redhat-developer/quarkus-ls/tree/master/qute.ls[Qute language server], already used in our popular  https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Quarkus Tools extension] for Visual Studio Code.
```

Comment:
Sentence seems a bit long when reading.

I would probably say "This extension is based on the LSP4MP, its Quarkus add-on, and the Qute language server. These are all used in our popular ..."

Reply:
done